### PR TITLE
Fix Ingest Pipline Error in SentinelOne Package with k8s Elastic Agent

### PR DESCRIPTION
**Pull request by ChriZzn**
## Problem
When using the SentinelOne Package with 'alert' data_stream enabled, the Pipeline will fail if the Integration is deployed on an Elastic Agent which runs within a Kubernetes Cluster, because the Fields are already set due to Fleet: 
- orchestrator.cluster.name
- orchestrator.namespace
- container.name
 
This will cause the 'rename' Processor to fail, i added the 'ignore_failure' to the processors which are causing the failure

## Checklist

- [x] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [x] I have verified that all data streams collect metrics or logs.
- [x] I have added an entry to my package's `changelog.yml` file.
- [x] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

### CODE REVIEW

**Review by efd6 — COMMENTED** [review]
Suggest instead that you use `if: ctx. … == null`, or both the conditional and the `ignore_failure: true`.

**Comment by ChriZzn** [comment]
> Suggest instead that you use `if: ctx. … == null`, or both the conditional and the `ignore_failure: true`.

Hi Dan,

i added and if condition to check if the field is null...

Regards Christoph

File: packages/sentinel_one/changelog.yml [hunk 1]
```diff
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "1.20.1"
+  changes:
+    - description: Fix Ingest Pipline Error in SentinelOne Package with k8s Elastic Agent.
+      type: bug
```
Comment (efd6) [inline packages/sentinel_one/changelog.yml:5]:
```suggestion
      type: bugfix
```

File: packages/sentinel_one/data_stream/alert/elasticsearch/ingest_pipeline/default.yml [hunk 2]
```diff
@@ -322,10 +322,14 @@ processors:
       field: json.containerInfo.name
       target_field: container.name
       ignore_missing: true
+      ignore_failure: true
+      if: ctx.container?.name != null
```
Comment (efd6) [inline packages/sentinel_one/data_stream/alert/elasticsearch/ingest_pipeline/default.yml:326]:
```suggestion
      if: ctx.container?.name == null
```

File: packages/sentinel_one/data_stream/alert/elasticsearch/ingest_pipeline/default.yml [hunk 3]
```diff
@@ -322,10 +322,14 @@ processors:
...
   - rename:
       field: json.kubernetesInfo.cluster
       target_field: orchestrator.cluster.name
       ignore_missing: true
+      ignore_failure: true
+      if: ctx.orchestrator?.cluster?.name != null
```
Comment (efd6) [inline packages/sentinel_one/data_stream/alert/elasticsearch/ingest_pipeline/default.yml:332]:
```suggestion
      if: ctx.orchestrator?.cluster?.name == null
```

File: packages/sentinel_one/data_stream/alert/elasticsearch/ingest_pipeline/default.yml [hunk 4]
```diff
@@ -342,6 +346,8 @@ processors:
       field: json.kubernetesInfo.namespace
       target_field: orchestrator.namespace
       ignore_missing: true
+      ignore_failure: true
+      if: ctx.orchestrator?.namespace != null
```
Comment (efd6) [inline packages/sentinel_one/data_stream/alert/elasticsearch/ingest_pipeline/default.yml:350]:
```suggestion
      if: ctx.orchestrator?.namespace == null
```

**Review by efd6 — APPROVED** [review]

**Pull request merged**